### DISCUSSION TITLE
Wholesale Market B2B

### DISCUSSION THREAD

**Discussion by Haniel5** [inspireui/support#5308] [Working on Features]
in our website ( woocommerce based) we are using two prices for single product( customer price and Wholesale Price) for this we using Wholesale Market plugin by CedCommerce. Please show the price in same way on the app.

**Comment by lakshmish2004** [comment]
Yes I need this feature

**Comment by zaifajs** [comment]
Please add this feature

**Reply by ssemma442** [reply]
When are you adding this feature, we need wholesale prices

**Reply by prabhu009** [reply]
I am very much needed this feature please add asap..

**Comment by ecomdeveloperbd** [comment]
Yes I need this feature

**Comment by minhcasi** [comment]
https://wholesalesuiteplugin.com/

**Comment by minhcasi** [comment]
Feature requests:

1.Set wholesale prices (different prices per product) on Simple product or Variable product variations

2. Show/hide the retail price from wholesale customers Private store/WooCommerce catalog mode option (hides price & add to cart until login)

3.Disable WooCommerce coupons for wholesale customers

4.Adjust the wholesale role & wholesale price text

5.Works with any theme

6.Works with WooCommerce CSV import/export tool

7.Multi-currency support with Aelia Currency Switcher, WPML Currency & WOOCS

8.Translatable via WPML Multi-site compatible

**Reply by ecomdeveloperbd** [reply]
1. Pre-Order and Call Price set or Ask Price set.
2. and Price set like

50pis price
100 pis price 
500 pis price set ... different different

**Comment by 3mmar3** [comment]
Ineed this feature important please

**Comment by ameenbadri7** [comment]
Ineed this feature important please

**Comment by 3mmar3** [comment]
6 Monthe To Add One A new addition ?

**Comment by minhcasi** [comment]
https://woocommerce.com/products/wholesale-for-woocommerce/
